.drone.star

```diff
@@ -76,7 +80,9 @@ def main(ctx):
             trigger = cron_trigger,
         )
 
-        gui_tests = gui_test_pipeline(ctx, trigger = cron_trigger)
+        gui_tests = gui_test_pipeline(ctx, trigger = cron_trigger) + \
+                    gui_test_pipeline(ctx, trigger = cron_trigger, server_version = "2.0.0-rc.1", server_type = "ocis")
```

Comment:
we should put that server version in a  variable where it will be easier to update and also add a tickbox to the release process of the client to update that

Reply:
where should I put the checkbox to update the server versions?